src/fft_small/sd_ifft.c

```diff
@@ -429,28 +603,28 @@ void CAT3(sd_ifft_basecase, m, 0)(const sd_fft_lctx_t Q, double* X, ulong j_mr,
 }
 
 EXTEND_BASECASE(4, 6)
+EXTEND_BASECASE(5, 7)
 EXTEND_BASECASE(6, 8)
+EXTEND_BASECASE(7, 9)
```

Comment:
There are a couple of lines that are unreached in the code coverage. Are you aware of any way to reach these lines? If you browse the files here in the commit, you will see where Codecov gives warnings.